subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak.c

```diff
@@ -21,12 +23,27 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_DECLARE(fp_storage, CONFIG_FP_STORAGE_LOG_LEVEL);
 
+/* Non-volatile bond information that is stored in the Settings subsystem. */
+struct fp_bond_info_nv {
+	uint8_t account_key_metadata_id;
+	uint8_t identity;
```

Comment:
I wonder if we actually need to store identity here. Isn't it already done by Bluetooth stack?

Reply:
It's needed when calling bt_unpair. Otherwise you would have to search through all identities with `bt_foreach_bond` to find the identity of the bond. Considering it's only 1 byte, we've decided it's easier to store it here.

Reply:
After the discussion we eventually decided not to store identity here.